topology/Purdue University/Purdue CMS/Purdue.yaml

```
@@ -269,6 +269,51 @@ Resources:
       StorageCapacityMax: 5.5
       StorageCapacityMin: 1
       TapeCapacity: 5.5
+  Purdue-Negishi:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          ID: b120c1c48f06940b0604f4a2803fde522d0d0595
+          Name: Stefan Piperov
+        Secondary:
+          ID: f11eeb608313b242e14f5aee602b3aff91839ace
+          Name: Erik Gough
+        Tertiary:
+          ID: c47d0e3b7d3a3de938261bd44ae692af60eedd4b
+          Name: Nick Smith
+      Security Contact:
+        Primary:
+          ID: b120c1c48f06940b0604f4a2803fde522d0d0595
+          Name: Stefan Piperov
+        Secondary:
+          ID: f11eeb608313b242e14f5aee602b3aff91839ace
+          Name: Erik Gough
+    Description: |-
+      Compute Element for Purdue Negishi cluster.
+      https://www.rcac.purdue.edu/compute/negishi
+    FQDN: osg.negishi.rcac.purdue.edu
+    ID: 929
```

The resource needs a new ID

```suggestion
    ID: 1416
```